src/app/App.tsx

```diff
@@ -69,6 +70,7 @@ export class App extends React.Component<Props> {
       LogBox.ignoreAllLogs(true)
     }
     await ValoraAnalytics.init()
+    enableScreens(true)
```

Comment:
This is already the default. If we want to keep it to be more explicit, it would be better to call it globally in this module instead of in `componentDidMount`.

https://github.com/software-mansion/react-native-screens/blob/9b3adea78a3dbc9a6930266cbd2da1b5235091a8/src/index.tsx#L19-L27

Reply:
Resolved in [02f51e2](https://github.com/valora-inc/wallet/pull/2669/commits/02f51e26325859a58dd767d05f561398b666a460)!